src/main/kotlin/uk/gov/dluhc/emsintegrationapi/mapper/PostalVoteMapper.kt

```diff
@@ -108,11 +109,13 @@ class PostalVoteMapper(private val instantMapper: InstantMapper) {
             RejectedReasons(
                 englishReason = RejectedReason(
                     notes = englishRejectionNotes,
-                    reasons = englishRejectionReasons?.toList()
+                    reasons = englishRejectionReasons?.toList(),
+                    reasonList = englishRejectedReasonItems?.toList()?.map { item -> RejectedReasonItem(item.electorReason, item.type, item.includeInComms) }
```

Comment:
`englishRejectionReasons` here is derivable from `englishRejectedReasonItems` (would just be `englishRejectedReasonItems?.toList().map { it.type }`) so should probably just get the both fields from `englishRejectedReasonItems` and remove `englishRejectionReasons` from the entity